torchao/prototype/low_bit_optim/adam.py

```diff
@@ -28,6 +28,10 @@ def __init__(
         self.block_size = block_size
         self.bf16_stochastic_round = bf16_stochastic_round
         self.is_adamw = is_adamw
+        # Store the IDs of parameters to exclude
+        self.exclude_low_bit_optim_params_ids = set(
+            id(p) for p in exclude_low_bit_optim_params
+        ) if exclude_low_bit_optim_params else set()
```

Comment:
I think you can hash tensor directly (it will use object id internally). PyTorch optimizer already hashes tensors when it uses params as keys in `self.state`.

Reply:
Can you try this? I don't think you need to use `id(p)` explicitly.